packages/next-codemod/transforms/__testfixtures__/update-router-api/use-router-push.input.js

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from 'next/router'
+
+function ActiveLink({ children, href, as }) {
+  const router = useRouter()
+  const style = {
+    marginRight: 10,
+    color: router.pathname === href ? 'red' : 'black',
+  }
+
+  const handleClick = (e) => {
+    e.preventDefault()
+    router.push(as, href)
```

Comment:
Seems this one should be the other way around 👀 
```suggestion
    router.push(href, as)
```

Reply:
Thanks ! I just update the naming ;)